proposals/0029-cooperative-vector.md

```diff
@@ -817,8 +817,8 @@ layout and dataype of the weight matrix from and to any of the layouts in
 enum D3D12_LINEAR_ALGEBRA_MATRIX_LAYOUT {
     D3D12_LINEAR_ALGEBRA_MATRIX_LAYOUT_ROW_MAJOR,
     D3D12_LINEAR_ALGEBRA_MATRIX_LAYOUT_COLUMN_MAJOR,
-    D3D12_LINEAR_ALGEBRA_MATRIX_LAYOUT_INFERENCING_OPTIMAL,
-    D3D12_LINEAR_ALGEBRA_MATRIX_LAYOUT_TRAINING_OPTIMAL
+    D3D12_LINEAR_ALGEBRA_MATRIX_LAYOUT_MUL_OPTIMAL,
+    D3D12_LINEAR_ALGEBRA_MATRIX_LAYOUT_OUTER_PRODUCT_OPTIMAL
```

Comment:
Double checking it's fine that the D3D caps queries still have the names D3D12_COOPERATIVE_VECTOR_PROPERTIES_TRAINING and _INFERENCE?

The D3D12_COOPERATIVE_VECTOR_PROPERTIES_TRAINING struct applies to both outer product accumulate and vector accumulate.  Perhaps it's fine that these caps structs are named around the scenario and how they show up in the D3D12_FEATURE_DATA_COOPERATIVE_VECTOR struct indicates to which sets of ops they apply to.

Reply:
D3D12_COOPERATIVE_VECTOR_PROPERTIES_MUL, D3D12_COOPERATIVE_VECTOR_PROPERTIES_ACCUMULATE?

Reply:
That sounds good!

Reply:
Updated